cmake/ccf_app.cmake

```diff
@@ -257,8 +257,6 @@ function(add_ccf_app name)
       ${virt_name} PRIVATE ${PARSED_ARGS_LINK_LIBS_VIRTUAL} ccf.virtual
     )
 
-    target_link_options(${virt_name} PRIVATE LINKER:--no-undefined)
```

Comment:
@letmaik Do you know why this was added? I guess to catch the versions of the OE libraries that did/didn't include the new symbol?

Reply:
I added this to be consistent in how OE links enclave images when linking against oeenclave. And yes, in this case during testing without it an unresolved symbol only lead to an error at runtime. So I think you should keep it for non-SAN builds.